request/graphql/schema/generate.go

```diff
@@ -365,14 +409,23 @@ func (g *Generator) buildTypes(
 		// will be reassigned before the thunk is run
 		collection := c
 		fieldDescriptions := collection.Schema.Fields
+		isEmbeddedObject := collection.Description.Name == ""
```

Comment:
suggestion: A brief note / concrete example here describing what is meant by an "embedded object" here would be nice.

Reply:
I do see `embedded` object as a ubiquitous database/defra concept that needs no explanation against every variable declaration that mentions it.  I'd be quite against doing this.